patches/v8/0022-Add-GetExternalMemory.patch

```diff
@@ -0,0 +1,49 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Kenton Varda <kenton@cloudflare.com>
+Date: Thu, 26 Mar 2020 19:15:48 -0500
+Subject: Add methods to get heap and external memory sizes directly.
+
+`GetHeapStatistics()` exists for this, but also collects a lot of other info and apparently performs non-trivial computation. These new accessors are intended to be very fast.
+
+Signed-off-by: James M Snell <jsnell@cloudflare.com>
+
+diff --git a/include/v8-isolate.h b/include/v8-isolate.h
+index 3a261b4ad1aaa8f424a2a7e5d6dbad7b4ad8d7bf..3d3781ed2d60f12d8ec70ea6091f32b07e4ae9ad 100644
+--- a/include/v8-isolate.h
++++ b/include/v8-isolate.h
```

Comment:
Why not just use isolate->GetHeapStatistics()?

Reply:
Can we improve the documentation on the patch for the exact reason and differences for posterity?

Reply:
For those reading just the comments here tho... `GetHeapStatistics()` actually does not provide the correct result here so adding the `GetExternalMemory()` patch is required to get the most accurate view that matches what we use internally.